### DISCUSSION TITLE
The UI looks beautiful

### DISCUSSION THREAD

**Discussion by olimorris** [nvim-neotest/neotest#27] [General]
I've hacked on some of the icons and added some highlight groups to my [theme](https://github.com/olimorris/onedarkpro.nvim)...it's looking beautiful:

![Screen Shot 2022-06-13 at 07 44 46@2x](https://user-images.githubusercontent.com/9512444/173295168-bff71608-15b9-4bf8-9a34-0f4695693907.png)

```lua
    icons = {
      expanded = "",
      child_prefix = "",
      child_indent = "",
      final_child_prefix = "",
      non_collapsible = "",
      collapsed = "",

      passed = "",
      running = "",
      failed = "",
      unknown = ""
    },
```

**Comment by rcarriga** [comment]
Looks great! A theming section in the wiki might actually be a nice idea. Think a lot of people would like to customise this stuff but especially newer users aren't used to it coming from other editors so would be good to give samples

**Comment by chanced** [comment]
@olimorris how are you able to achieve line spacing like that without negatively impacting the nerdfont appearance?

**Reply by olimorris** [reply]
That seems to be the joy of using Kitty. I was using Alacritty prior but the vertical spacing was really difficult to get right

**Reply by chanced** [reply]
Heh, I just gave up on Alacritty due to font rendering being horrific on a Mac and switched back to iterm2.

I'll give Kitty a try. Thanks.

**Reply by olimorris** [reply]
My Kitty config is [here](https://github.com/olimorris/dotfiles/blob/main/.config/kitty/kitty.conf) if at all helpful

**Reply by chanced** [reply]
Yep, that did it! Thanks!
